### ISSUE TITLE
When using stack limit 1, new items dont appear if bags already opened.

### ISSUE THREAD

**Issue by strickland** [arkayenro/arkinventory#1315]
Simply I'm using stack limit 1, to make all empty slots appear as one empty icon, then if my bags already opened and I received new items, they don't appear until I either refresh or close+open bags, because I have also been using the Sorting option as "On window open" 

Reason why i'm using on window open, is because if i'm mass-selling stuff from the windows, stuff keep moving around and I can't click quick on stuff, you can try that by selling items/equipment to the vendor, if you have the option instantly sorting on, things will keep moving around and you need to wait 1sec between each sale to find the next item.

So basically two issues i'm having in one, don't know whats the best solution for this.

**Comment by arkayenro** [comment]
use instant sorting
click on the refresh button when needed
close/open the bag when needed

it cant sort both instantly and on window open, you have to pick the one that works better for you
